### DISCUSSION TITLE
getGroupBW

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#1298] [Questions / Documentation]
Hello,
I have some TFBS that I want to see how the accessibility changes across the cell types (groups) in those specific locations. 

I used getGroupBW with default parameter, and the bigWig visualization in IGV looks like this(for two of the groups):
<img width="1056" alt="Screen Shot 2022-02-18 at 10 22 24 PM" src="https://user-images.githubusercontent.com/19545198/154755448-f299b508-5c12-44ba-ba95-9c08f6a26537.png">
I was expecting to get something similar to the plotBrowserTrack output - I was wondering if you could share the difference between the bigWig visualization and the plotBrowserTrack visualization? is there a way to extract the plotBrowserTrack bed file?

Thank you very much,

-Rojin

**Comment by rcorces** [comment]
its just a matter of scale of the y axis in igv.
igv doesnt handle y axis scaling very well. you can turn on autoscale or fix your scale to something more informative.
